config/locales/views/users/en.yml

```diff
@@ -91,4 +91,6 @@ en:
       see_all : "See All"
       fork_notification: "%{user} forked your Project %{project}"
       star_notification: "%{user} starred your Project %{project}"
-      new_assignment_notification: "You have been assigned a new assignment '%{assignment_name}'"
\ No newline at end of file
+      new_assignment_notification: "You have been assigned a new assignment '%{assignment_name}'"
+      forum_thread_notification: "%{user} posted a new thread '%{thread}'"
+      comment_notif: "%{user} commented '%{mssg}' on thread '%{thread}'"
```

Comment:
```suggestion
      comment_notif: "%{user} commented on thread '%{thread}'"
```

The message could be large. Let's keep it just to notify you if there is a new comment on the thread!

Reply:
Currently its truncated to merely 4 words [permlink](https://github.com/CircuitVerse/CircuitVerse/pull/3483/files#diff-2ba232ca3549893005e0136addf5f50454622b5f49ddfc0bf88d98f7d2d603feR10).
Is that limit good or should I change it? WDYT @vedant-jain03

Reply:
I think that notifying users regarding events is the primary task, WDYT?
What if the thread heading is so large?

Reply:
Yes you are right, Thread heading should be truncated too... because questions would be generally longer.

For mssg, it would look something like this. (adding screenshots... will update in next commit).
![no-comment](https://user-images.githubusercontent.com/85568177/212107716-5551a459-bf55-4416-9347-60e5d16a31a4.jpg)

![comment](https://user-images.githubusercontent.com/85568177/212107704-ea6e82c9-0bde-4ed9-9f19-69cd2e1a6539.jpg)